src/index.js

```diff
@@ -15,4 +14,8 @@ form.addEventListener('submit', (e) => {
   addList(namevalue, scorevalue);
   inputName.value = '';
```

Comment:
Your project looks great. Kindly ensure that user data reflects on the scoreboard when saved.

Reply:
Hello @emmiiorji 
Thank you for your suggestion.
Please check the project's instructions. According to that, the DOM is updated using the refresh button when it fetches data from the API. The submit button is for saving the data into API.